doc/nrf/device_guides/wifi_coex.rst

```diff
@@ -242,6 +242,71 @@ To enable the generic three-wire coexistence, do the following:
    * :kconfig:option:`CONFIG_MPSL_CX`
    * :kconfig:option:`CONFIG_MPSL_CX_3WIRE`
 
+.. _ug_radio_mpsl_cx_generic_1wire:
+
+Generic one wire coexistence
+============================
+
+Refer to :ref:`ug_radio_coex_mpsl_cx_based` for the general requirements of this implementation.
+
+The generic one wire coexistence interface can be used to allow the :ref:`SoftDevice Controller <nrfxlib:softdevice_controller>` to coexist alongside an LTE device on a separate chip.
+It is specifically designed for the coex interface of the nRF91 Series SiP.
```

Comment:
I'm a bit confused by the title being generic one wire interface, but this documentation saying it is for LTE devices and is specifically designed for the coex interface of the nRF91 series.

Could it be an idea to split it as is done for the generic 3 wire and nrf700x coex interface?

Reply:
We can rephrase for instance to say that it's "compatible with the coex interface for the nrf91". I just wanted to make sure customers aren't confused when we deprecate the other one